simple.c

```diff
@@ -119,8 +119,8 @@ bool Simple_SetStringProperty(void *SimpleContext, char *Property) {
     SimpleStruct *Simple = (SimpleStruct *)SimpleContext;
     char OldProperty[320] = {0};
 
-    String_CopyLength(OldProperty, Simple->StringProperty, 320);
-    String_CopyLength(Simple->StringProperty, Property, 320);
+    strncpy(OldProperty, Simple->StringProperty, sizeof(OldProperty));
+    strncpy(Simple->StringProperty, Property, sizeof(OldProperty));
```

Comment:
`strcpy_s` and not `_CRT_SECURE_NO_WARNINGS`?

Reply:
Ok it should be fixed now.